elife/goaws.sls

```diff
@@ -0,0 +1,31 @@
+goaws-install:
+    cmd.run:
+        - name: go get github.com/p4tin/goaws
+        - env:
+            - GOPATH: /usr/local
+        - require:
+            - pkg: golang-go
+            - aws-cli
```

Comment:
how long does this take if goaws had already been downloaded/installed? If it's more than a second we should add an `- unless:` to avoid executing the command when not needed